integration-sqs-to-lambda/Function.cs

```diff
@@ -0,0 +1,70 @@
+using Amazon.Lambda.Core;
+using Amazon.Lambda.SQSEvents;
+using static Amazon.Lambda.SQSEvents.SQSBatchResponse;
+
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace SqsIntegration;
+
+public class Function
+{
+
+    private readonly List<BatchItemFailure> _batchItemFailures;
+
+    /// <summary>
+    /// Default constructor. This constructor is used by Lambda to construct the instance. When invoked in a Lambda environment
+    /// the AWS credentials will come from the IAM role associated with the function and the AWS region will be set to the
+    /// region the Lambda function is executed in.
+    /// </summary>
+    public Function()
+    {
+        _batchItemFailures = new List<BatchItemFailure>();
```

Comment:
On the second invoke this same data structure will be re-used as the constructor wont run again once the execution environment is warm. Maybe move the initialization to the FunctionHandler so that it is empty on every invoke.

Reply:
added comment for that